pipenv/project.py

```diff
@@ -416,17 +416,25 @@ def get_name(name, location):
     @property
     def virtualenv_name(self):
         # type: () -> str
+        custom_name = os.getenv("PIPENV_CUSTOM_VENV_NAME")
+        if custom_name:
+            return custom_name
+        names = []
```

Comment:
I don't think we should get into the business of having a list of names to join here.

Reply:
I can do that differently.  Anytime I have more than 2 items to join with the same separator I use a join() as a habit.